topics/go/concurrency/data_race/example1/example1.go

```diff
@@ -27,7 +27,7 @@ func main() {
 	// Create two goroutines.
 	for i := 0; i < grs; i++ {
 		go func() {
-			for count := 0; count < 2; count++ {
+			for x := 0; x < 2; x++ {
```

Comment:
How about we use `i` for this variable and `g` for the variable in the first loop?

We need to update the README with a new playground version of this code.

Reply:
Fair point. Also thanks for reminding me about the README links. Pushed an amended commit to address this.